test/Microsoft.DotNet.Docker.Tests/DockerHelper.cs

```diff
@@ -44,7 +44,7 @@ public void DeleteImage(string tag)
 
         public void DeleteVolume(string name)
         {
-            if (ResourceExists("volume", name))
+            if (VolumeExists(name))
```

Comment:
I can't see how introducing `VolumeExists` makes things better.  Since this can be achieved with `ResourceExists("volume", $"-f \"name={name}\"")`

Reply:
I think it is more readable and provides symmetry with the DeleteImage implementation.  Leaving as is.